CorsixTH/Lua/entities/humanoid.lua

```diff
@@ -341,6 +345,18 @@ function Humanoid:afterLoad(old, new)
   if old < 134 and new >= 134 then
     self.staff_change_callbacks = {}
   end
+  if old < 210 then
+    -- We renamed the old sad7 to sad2; and sad2 to dying1. Make sure we adjust
+    -- sad2 to the new mood
```

Comment:
Should be `sad2 and sad7`

Reply:
Euhm, sad3 to 7 are no dying1 to 5 as well.

Reply:
Oops my bad!